### DISCUSSION TITLE
Unable to Connect - Project Status

### DISCUSSION THREAD

**Discussion by mztmoraes** [supabase/supabase#23728] [Questions]
Hi community

I've been disconnected from my project for hours. The status shows "Unable to Connect".

I have the pro plan, and I've already opened a few tickets but no response.

My app is down because of this.

What to do?
Is there any way to fix it with supabase in these critical cases?

Or should I pray?

Thanks any help

**Comment by pdovhomilja** [comment]
Hi, same issue. Did you find any solution?
